internal/http/services/owncloud/ocs/handlers/apps/sharing/shares/export_test.go

```diff
@@ -0,0 +1,3 @@
+package shares
+
+var GetUnmountedShares = getUnmountedShares
```

Comment:
What kind of black magic is this? What are we trying to achieve here?

Reply:
i dont want to make the getUnmountedShares public, so... just expose it for the test which is not part of the artifact later on, similar to the go core:

https://github.com/golang/go/blob/master/src/fmt/export_test.go